### PR TITLE
Developer docs structure with improved navigation

### PR DESCRIPTION
**Pull request by benjaoming**
### Summary

Fixes very broken navigation in developer docs

screen shot 2017-12-19 at 17 51 03
screen shot 2017-12-19 at 17 51 14

### Reviewer guidance

This is built automatically with the usual crazy amount of Warnings that I'm working through ATM... I'm contemplating adding another PR building on top of this right meow, so it's not so important if there are errors here... we just need to add in missing chapters and get the navigation working again.

Maybe a suggestion for the intro text?

### References

#2828 
#2662

----

### Contributor Checklist

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] Contributor has fully tested the PR manually
- [x] Screenshots of any front-end changes are in the PR description
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: docs-developer/index.rst [hunk 1]
```diff
@@ -1,47 +1,25 @@
 Developer Guide
 ===============
 
-Start here
-----------
+Please first of all, read :ref:`our gettting started guide <getting_started>`.
```
Comment (jamalex) [inline docs-developer/index.rst]: "First of all, please read..."

File: docs-developer/index.rst [hunk 2]
```diff
@@ -1,47 +1,25 @@
...
+to learn   about :ref:`setting up your development environment <dev_env_intro>`,
```
Comment (jamalex) [inline docs-developer/index.rst]: Linting: delete extra space

File: docs-developer/index.rst [hunk 3]
```diff
@@ -1,47 +1,25 @@
...
+:ref:`a nice development workflow <workflow_intro>` and :ref:`our stack of technology <stack>`.
```
Comment (jamalex) [inline docs-developer/index.rst]: "our technology stack"

Comment (jamalex) [reply docs-developer/index.rst]: (otherwise I just imagine the pile of old tablets in our cabinet...)

File: docs-developer/index.rst [hunk 4]
```diff
@@ -1,47 +1,25 @@
...
 
-.. toctree::
-  :maxdepth: 1
-
-  getting_started
-  stack
+If you are looking for help for installing, configuring and using Kolibri,
```
Comment (jamalex) [inline docs-developer/index.rst]: Remove second "for"

File: docs-developer/index.rst (hunk 3 above)
Comment (benjaoming) [reply docs-developer/index.rst]: hahaha, will add a chapter about that? :D

**Comment by indirectlylit** [comment]
@benjaoming broken stuff still looks the same to me after this PR:

image

http://kolibri-dev.readthedocs.io/en/develop/

**Comment by benjaoming** [comment]
@indirectlylit not sure I understand the screenshot, could you elaborate what "broken stuff"?

**Comment by indirectlylit** [comment]
hahha I was literally just copying what you wrote with the same screenshot:

image

By your comment and screenshot I had assumed you meant all the issues with the first two paragraphs:

* typo in "our gettting started guide"
* extra period
* two links that go to essentially the same page
* awkward overall wording of the first paragraph
* broken link in second paragraph

**Pull request merged**